incubator/hnc/pkg/forest/forest.go

```diff
@@ -53,6 +63,10 @@ type Forest struct {
 	// We can also move the lock out of the forest and pass it to all reconcilers that need the lock.
 	// In that way, we don't need to put the list in the forest.
 	types []TypeSyncer
+
+	// config is the ConfigReconciler that an object reconciler can call if the status of the HNCConfiguration
+	// object needs to be updated.
+	Config NumPropagatedObjectsSyncer
```

Comment:
I'd suggest changing Config to something like ObjectStatsSyncer

Reply:
Sure! Thanks for the suggestion!